src/Queries/LatestMollieWebhookCallByResourceId.php

```diff
@@ -22,8 +21,8 @@ public function find(
             ->where('id', '!=', $ignoreWebhookCall->getKey())
             ->where('name', MollieSignatureValidator::NAME)
             ->where('payload', 'LIKE', "%\"id\":\"{$resourceId->value()}\"%")
-            ->when($payloadFragment, Closure::fromCallable([$this, 'filterByPayloadFragment']))
-            ->where(Closure::fromCallable([$this, 'ignoreFailedWebhookCalls']))
+            ->when($payloadFragment, ($this->filterByPayloadFragment(...))(...))
+            ->where(($this->ignoreFailedWebhookCalls(...))(...))
```

Comment:
I think you can simply do:
```suggestion
            ->when($payloadFragment, $this->filterByPayloadFragment(...))
            ->where($this->ignoreFailedWebhookCalls(...))
```